packages/e2e-tests/specs/editor/various/reusable-blocks.test.js

```diff
@@ -260,7 +235,7 @@ describe( 'Reusable blocks', () => {
 		// Save the reusable block
 		await page.click( publishButtonSelector );
 		await page.waitForXPath(
-			'//*[contains(@class, "components-snackbar")]/*[text()="Reusable Block updated."]'
+			'//*[contains(@class, "components-snackbar")]/*[text()="Reusable block updated."]'
```

Comment:
This breaks due to this: https://github.com/WordPress/gutenberg/blob/master/lib/compat.php#L266

Since we make `blocks` with a lower `b`, should we change the other Labels as well?

Reply:
It seems there's an inconsistency between Core and Gutenberg. Core menus favor capitalizing everything and Gutenberg the opposite. This is one area where the same string can be used in two different places. The issue is that at some point the guideline here should be consistent between Core and  Gutenberg but it's a broader issue.